tests/foreman/cli/test_satellitesync.py

```diff
@@ -1165,9 +1166,8 @@ def test_negative_export_cv_with_puppet_repo(self):
         self.assert_error_msg(
             error,
             "Could not export the content view:\n  "
-            "Error: The Content View '{}' contains Puppet modules, "
-            "this is not supported at this time. Please remove the modules, "
-            "publish a new version and try the export again.\n".format(content_view['name'])
+            "Error: Ensure the content view version "
+            "'{} {}' has at least one repository.\n".format(content_view['name'], cv_version)
```

Comment:
@latran Is this error specific to yum repository? 

You missed updating these lines in the docstring, please take care of it  https://github.com/SatelliteQE/robottelo/pull/7547/files#diff-946a6c123047c6ea6b245a6a4e3a0c00R1134-R1138

Reply:
Fixed the expected result.